java7/java.l

```diff
@@ -98,13 +98,13 @@ instanceof INSTANCEOF
 \|= OREQ
 
 [0-9]+\.[0-9]*([eE][+-]?[0-9]+)?[fFdD]?|\.[0-9]+([eE][+-]?[0-9]+)?[fFdD]?|[0-9]+[eE][+-]?[0-9]+[fFdD]?|[0-9]+[fFdD] FLOATING_POINT_LITERAL
-(0_[0-7]+|0b[01]+|0x[0-9A-Fa-f]+|[0-9]+)[lL]? INTEGER_LITERAL
+(0_[0-7]+|[0-9][0-9_]+[0-9]|0b[01]+|0[xX][0-9A-Fa-f]+|[0-9]+)[lL]? INTEGER_LITERAL

```

Comment:
Shouldn't this be `0[bB][01]+`?

Reply:
Good spot. I only fixed the ones that flagged up when parsing the standard library.